src/main/java/org/broadinstitute/hellbender/tools/walkers/featuremapping/FlowFeatureMapper.java

```diff
@@ -759,5 +886,35 @@ private FeatureMapper buildMapper() {
             throw new GATKException("unsupported mappingFeature: " + fmArgs.mappingFeature);
         }
     }
+
+    private FlowBasedRead buildTrimmedFlowBasedRead(final GATKRead read, final TrimInfo trimInfo, final FlowBasedReadUtils.ReadGroupInfo rgInfo) {
```

Comment:
I think that this code might be duplicated. 
HaplotypeCaller routinely trims reads. Could you check if there is a function like that?

Reply:
Really sorry for being a pain, can you explain why functions from ReadClipper (e.g. SoftClipByReadCoordinates) etc. can't be used for this task?